spec/system/kudo_spec.rb

```diff
@@ -19,6 +19,9 @@
     fill_in 'Password', with: employee.password
     click_button 'Log in'
     expect(page).to have_content 'Signed in successfully.'
+    within('#Kudo_Points') do
```

Comment:
it's fine, but please read a little about css naming conventions or finding a HTML element by `data-test-id` attribute instead of ID

Reply:
I replaced id with data-test-id, in fact now it is immediately clear for what purpose it is introduced :)